samples/detect.v2beta1.js

```diff
@@ -0,0 +1,822 @@
+
+function createKnowledgeBase(projectId, displayName) {
+  // [START dialogflow_create_knowledge_base]
+  // Imports the Dialogflow client library
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow client.
+  var client = new dialogflow.KnowledgeBasesClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  //const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
+  //const displayName = `your knowledge base display name, e.g. myKnowledgeBase`;
+
+  var formattedParent = client.projectPath(projectId);
+  var knowledgeBase = {
+    displayName: displayName,
+  };
+  var request = {
+    parent: formattedParent,
+    knowledgeBase: knowledgeBase,
+  };
+
+  client
+    .createKnowledgeBase(request)
+    .then(responses => {
+      const result = responses[0];
+      console.log(`Name: ${result.name}`);
+      console.log(`displayName: ${result.displayName}`);
+      return result.displayName;
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END dialogflow_create_knowledge_base]
+}
+
+function getKnowledgeBase(projectId, knowledgeBaseFullName) {
+  // [START dialogflow_get_knowledge_base]
+  // Imports the Dialogflow client library
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow client.
+  var client = new dialogflow.KnowledgeBasesClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  //const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
+  //const knowledgeBaseFullName = `the full path of your knowledge base, e.g my-Gcloud-project/myKnowledgeBase`;
+
+  const fullName = knowledgeBaseFullName;
+
+  client
+    .getKnowledgeBase({
+      name: fullName,
+    })
+    .then(responses => {
+      const result = responses[0];
+      console.log(`Name: ${result.name}`);
+      console.log(`displayName: ${result.displayName}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END dialogflow_get_knowledge_base]
+}
+
+function listKnowledgeBases(projectId) {
+  // [START dialogflow_list_knowledge_base]
+  // Imports the Dialogflow client library
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow KnowledgeBasesClient.
+  var client = new dialogflow.KnowledgeBasesClient({
+    projectPath: projectId,
+  });
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  //const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
+
+  var formattedParent = client.projectPath(projectId);
+
+  client
+    .listKnowledgeBases({parent: formattedParent})
+    .then(responses => {
+      var resources = responses[0];
+      resources.forEach(r => {
+        console.log(`  Display name is : ${r.displayName}`);
```

Comment:
nit: remove leading whitespace

```js
`  Display name...` // --> `Display name...`
```

As always, please apply to the whole diff & check whitespace for extraneous leading/trailing spaces or extraneous newlines

Reply:
**Not a blocker** – I see lots of this, likely cause Python folks did this – no worries for this PR